src/app/components/lineage-graph/lineage-graph.component.ts

```diff
@@ -11,10 +11,15 @@ import {
 } from "@angular/core";
 import { Edge, MiniMapPosition } from "@swimlane/ngx-graph";
 import { ClusterNode, Node } from "@swimlane/ngx-graph/lib/models/node.model";
+import { Observable } from "rxjs";
+import { AccountDetailsFragment } from "src/app/api/kamu.graphql.interface";
+import AppValues from "src/app/common/app.values";
+import { AccountService } from "src/app/services/account.service";
 
 @Component({
     selector: "app-lineage-graph",
     templateUrl: "./lineage-graph.component.html",
+    styleUrls: ["./lineage-graph.component.sass"],
```

Comment:
It seems that the styles file is empty, it's really needed?

Reply:
Removed